### PR TITLE
chore: use `@exodus/eslint-config`

### PR DESCRIPTION
**Pull request by sparten11740**
Use `@exodus/eslint-config` and run lints in the ci

Towards #9

### CODE REVIEW

**Review by ChALkeR — CHANGES_REQUESTED** [review]
nACK

This is a public repo.

File: create-fetchival.js [hunk 1]
```diff
@@ -20,15 +20,15 @@ function createFetchival({ fetch }) {
       headers: {
         Accept: 'application/json',
         'Content-Type': 'application/json',
-        ...(opts.headers || {}),
+        ...opts.headers,
```
Comment (ChALkeR) [inline create-fetchival.js:23]: nACK, this is breaking

**Comment by sparten11740** [comment]
see my comment https://github.com/ExodusMovement/fetch/issues/9#issuecomment-1717199823, I disagree. Especially because this is a public repo there should be tooling that enforces standards.

**Comment by sparten11740** [comment]
the deps are now open-sourced and I removed the private registry access in the CI: https://github.com/ExodusMovement/fetch/pull/11/commits/c7a5e465c9cdd59a9302818c1d3339f1e543430b

**Pull request merged**